### ISSUE TITLE
after installing this package, plan(multisession) would simply never stop running

### ISSUE THREAD

**Issue by PedroMilanezAlmeida** [HenrikBengtsson/BiocParallel.FutureParam#6]
As described in the title. Important, before installing BiocParallel.FutureParam, ```plan(multisession)``` after ```library(future.apply)``` had no problems at all.

Weirdly, standalone R and R in the terminal have no issues, problem found only if using Rstudio.

I uninstalled Rstudio/R/all libraries and just reinstalled everything and, even on a clean installation, the chunk below never ends:

```
install.packages("future.apply")
library(future.apply)
plan(multisession)
```

In the sessionInfo of R standalone vs Rstudio, the only difference I can see is in BLAS (see below), but I dont know if that means anything.

For R standalone:
```
> sessionInfo()
R version 4.1.0 (2021-05-18)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] future.apply_1.7.0 future_1.21.0     

loaded via a namespace (and not attached):
[1] compiler_4.1.0    parallelly_1.26.0 tools_4.1.0       parallel_4.1.0    listenv_0.8.0     codetools_0.2-18  digest_0.6.27    
[8] globals_0.14.0   

```
For R in Rstudio:
```
> sessionInfo()
R version 4.1.0 (2021-05-18)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Mojave 10.14.6

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] future.apply_1.7.0 future_1.21.0     

loaded via a namespace (and not attached):
[1] compiler_4.1.0    parallelly_1.26.0 parallel_4.1.0    tools_4.1.0       listenv_0.8.0     codetools_0.2-18 
[7] digest_0.6.27     globals_0.14.0   
```

Any help would be much appreciated.

**Comment by HenrikBengtsson** [comment]
Hi. I suspect what happened is that **parallelly** got updated too at the same time.  parallelly 1.26.0 was released on 2021-06-09 and it gained new powers where it can _set up_ the workers concurrently, cf. https://www.jottr.org/2021/06/10/parallelly-1.26.0/.  Unfortunately, it turned out that there's a bug in R that causes this to fail in RStudio.  If you wait long enough, I think you should get a timeout.

For now, install:

```r
remotes::install_github("HenrikBengtsson/parallelly@1.26.0-9001")
```

That will work around the bug in R when running in RStudio.  The bug has been fixed in R devel, and I hope they'll fix it for R 4.1.1 too.

Let me know if this helps.

**Comment by PedroMilanezAlmeida** [comment]
Thank you very much for your quick reply!

The following command helped bring things back to normal indeed:

```
remotes::install_github("HenrikBengtsson/parallelly@1.26.0-9001")
```

Thanks again, I appreciate your help!

**Comment by HenrikBengtsson** [comment]
Great to hear

**Comment by HenrikBengtsson** [comment]
BTW, I checked the R core logs; the underlying bug has been fixed for R 4.1.1.
